### DISCUSSION TITLE
Custom extensions

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6589] [Ideas]
As you may know, my main need is building customized copies of turbowarp so that all my students can share the same view/experience without having to enable/disable addons by themselves. That is why I proposed (and very quickly obtained: thanks a lot ScratchAddons developers!!!!) to add an import/export option to the addon manager. 

Following this line, it would be important to me, when I need to add a custom estension(s), to be able to add this extension(s) to the "Automatically add extensions" addon, so that my students will find the extension preloaded as soon as they open Scratch (or TurboWarp). I think a list of boxes where you can add the URLs of the relevant extensions could work?

**Comment by GarboMuffin** [comment]
let's move this to https://github.com/TurboWarp/scratch-gui/issues/809

**Comment by sfederici** [comment]
I noticed the "documentation" button right now.  If it is not new, I hadn't noticed it before. It would be incredibly useful for every extension.

**Reply by GarboMuffin** [reply]
It's new, added at the same time as sample projects. Somebody needs to write the documentation first, which is a bit of hurdle

**Comment by sfederici** [comment]
Yes, of course. That is why I asked for sample projects first
